### PR TITLE
add id_panl_bppt, a parallel corpus for en-id

### PR DESCRIPTION
**Pull request by cahya-wirawan**
Parallel Text Corpora for English - Indonesian

### CODE REVIEW

File: datasets/id_panl_bppt/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- en
+- id
+licenses: []
```
Comment (lhoestq) [inline datasets/id_panl_bppt/README.md:9]:
```suggestion
licenses:
- unknown
```

File: datasets/id_panl_bppt/README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+multilinguality: 
+- translation
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
```
Comment (lhoestq) [inline datasets/id_panl_bppt/README.md:19]:
```suggestion
task_categories:
- conditional-text-generation
task_ids:
- machine-translation
```

File: datasets/id_panl_bppt/README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+- `id`: id of the sample
+- `translation`: the parallel sentence english-indonesian
+- `topic`: the topic of the sentence
+
+```
```
Comment (lhoestq) [inline datasets/id_panl_bppt/README.md:81]:
```suggestion
```

File: datasets/id_panl_bppt/README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+Indonesian
+
+## Dataset Structure
+
+[More Information Needed]
+
+### Data Instances
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/id_panl_bppt/README.md:74]: could you add an example from the dataset here ?

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for example

File: datasets/id_panl_bppt/README.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+### Data Fields
+- `id`: id of the sample
+- `translation`: the parallel sentence english-indonesian
+- `topic`: the topic of the sentence
```
Comment (lhoestq) [inline datasets/id_panl_bppt/README.md:79]: can you give more details about the values that `topic` can take ?

File: datasets/id_panl_bppt/id_panl_bppt.py [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+            src_tag="en",
+            tgt_tag="id",
+            topics=[
+                {"name": "Economy", "words": "150K"},
+                {"name": "International", "words": "150K"},
+                {"name": "Science", "words": "100K"},
+                {"name": "Sport", "words": "100K"},
+            ],
+        ),
+    ]
```
Comment (lhoestq) [inline datasets/id_panl_bppt/id_panl_bppt.py:79]:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = IdPanlBpptConfig
```
we need that to specify that we're using a custom config here

**Review by lhoestq — COMMENTED** [review]
Very nice thanks !

I left a few comments

**Comment by cahya-wirawan** [comment]
Hi @lhoestq, thanks for the review. I will have a look and update it accordingly.

**Comment by cahya-wirawan** [comment]
Strange error message :-)

```
>           tf_context = tf.python.context.context()  # eager mode context
E           AttributeError: module 'tensorflow' has no attribute 'python'
```

File: datasets/id_panl_bppt/README.md (hunk 5 above)
Comment (cahya-wirawan) [reply datasets/id_panl_bppt/README.md]: a topic list has been added

File: datasets/id_panl_bppt/README.md (hunk 4 above)
Comment (cahya-wirawan) [reply datasets/id_panl_bppt/README.md]: I just added an example of the dataset

**Review by lhoestq — APPROVED** [review]
Thanks !

Looks all good now :)

**Pull request merged**